sqlite-android/src/main/jni/sqlite/Android.mk

```diff
@@ -26,7 +26,13 @@ sqlite_flags := \
 	-DSQLITE_ENABLE_RTREE=1 \
 	-DSQLITE_OMIT_BUILTIN_TEST \
 	-DSQLITE_OMIT_COMPILEOPTION_DIAGS \
-	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600
+	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600 \
+	-DSQLITE_DQS=0 \
```

Comment:
This is a breaking change I believe. It may not be used by anyone (it is described as a "mis-feature" that the sqlite author regrets ever implementing) but still did enable string double-quoting, and disabling it alters semantics vs previous releases

Reply:
https://www.sqlite.org/compile.html#dqs

Reply:
Hmm, yes. I think I can remove this. It is not a feature, that really affects performance, but can break smth